recipes/fermipy/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set version = "0.10.4" %}
+
+package:
+  name: fermipy
+  version: {{version}}
+
+source:
+  fn: fermipy-{{version}}.tar.gz
+  url: https://pypi.io/packages/source/f/fermipy/fermipy-{{ version }}.tar.gz 
+  md5: 3eee87e3d2f324fdc51afe03f9d086a5
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [win]
```

Comment:
I believe all of the python dependencies are available on conda-forge.  As I mentioned above this package also depends on a number of C and C++ libraries (e.g. GSL, FFTW, and CFITSIO).  Eventually I'd like to get to the point where all of these could be installed with conda but for the moment I'm just focusing on getting the python installation working.